tests/GridFS/UnusableStream.php

```diff
@@ -20,7 +20,7 @@ public static function register($protocol = 'unusable'): void
             stream_wrapper_unregister($protocol);
         }
 
-        stream_wrapper_register($protocol, static::class, STREAM_IS_URL);
+        stream_wrapper_register($protocol, self::class, STREAM_IS_URL);
```

Comment:
`static` is useless for a `final` class. `self` can be resolved at compile time.